lightning/src/ln/monitor_tests.rs

```diff
@@ -516,7 +516,7 @@ fn do_test_claim_value_force_close(prev_commitment_tx: bool) {
 			claimable_amount_satoshis: 4_000,
 			timeout_height: htlc_cltv_timeout,
 		}],
-	nodes[1].chain_monitor.chain_monitor.monitors.read().unwrap().get(&funding_outpoint).unwrap().get_claimable_balances());
+		nodes[1].chain_monitor.chain_monitor.get_monitor(funding_outpoint).unwrap().get_claimable_balances());
```

Comment:
extra tab

Reply:
Hmm, its a parameter to the `assert_eq` at the same indentation - shouldn't it be indented?